docs/MAINTAINERS.adoc

```diff
@@ -155,6 +158,9 @@ Code reviews do NOT pursue the following goals:
   Not everything is going to be merged, and reviewers are expected to focus on the Jenkins ecosystem integrity first.
   We guide contributors and help them to get their changes integrated, but it needs cooperation on both sides.
   It is **fine** to close invalid and inactive pull requests if there is no activity by a submitter or other contributors.
+  When a pull-request has no activities in the last 30days, it can be marked with the label `stalled`.
+  If there is no consensus or activities after 30 more days, the pull-request can be marked as `proposed-for-close`.
+  The pull-request will then be closed in the next 96hrs with no new activities.
```

Comment:
```suggestion
  When a pull request remains inactive for a month, it can be marked with the label `stalled`.
  If the pull request remains inactive or without consensus for yet another month, the pull request can be marked as `proposed-for-close`.
  The pull request should then be closed in approximately a week if this state persists.
```